modules/mobile/app/controllers/mobile/v0/decision_letters_controller.rb

```diff
@@ -5,11 +5,21 @@
 module Mobile
   module V0
     class DecisionLettersController < ApplicationController
+      before_action { authorize :bgs, :access? }
```

Comment:
I think we need to limit this to only the download action.

Reply:
I think we should be consistent between both controllers on this. if there isn't a participant id and the index uses the ssn as a fallback, then the user should still be able to use download the letter from index.

Reply:
I defer to you guys on this one since you know more about it than I do. I can see your point about it exposing the download url. Not sure if they'd actually be able to use that url without going through the vets-api auth though.